app/src/main/res/values/styles.xml

```diff
@@ -1,11 +1,11 @@
 <resources>
 
     <!-- Base application theme. -->
-    <style name="AppTheme" parent="Theme.AppCompat.Light.NoActionBar">
-        <!-- Customize your theme here. -->
-        <item name="colorPrimary">@color/colorPrimary</item>
-        <item name="colorPrimaryDark">@color/colorPrimaryDark</item>
-        <item name="colorAccent">@color/colorAccent</item>
+    <style name="AppTheme" parent="Theme.Material3.DayNight.NoActionBar">
```

Comment:
@luongvo I think we can use this theme `android:Theme.Material.Light.NoActionBar` So, we don't add `com.google.android.material:material`
```suggestion
    <style name="AppTheme" parent="android:Theme.Material.Light.NoActionBar">
```

Reply:
@Wadeewee `Theme.Material3.DayNight.NoActionBar` does not support dark background properly before having compose background rendering.

Reply:
@Wadeewee After checked again, we can remove it and use the current style.